src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs

```diff
@@ -66,7 +66,18 @@ public partial class MergePlugin : IConsensusWrapperPlugin, ISynchronizationPlug
         public string Description => "Merge plugin for ETH1-ETH2";
         public string Author => "Nethermind";
 
-        public virtual bool MergeEnabled => _mergeConfig.Enabled;
+        public virtual bool MergeEnabled => HasTtd(_api)
+                                            && !IsPreMergeConsensusAuRa(_api); // AuRa has dedicated plugin AuRaMergePlugin
+
+        protected bool HasTtd(INethermindApi api)
+        {
+            return api.SpecProvider?.TerminalTotalDifficulty != null || api.Config<IMergeConfig>().TerminalTotalDifficulty != null;
```

Comment:
why not `_mergeConfig` ?

Reply:
null reference, but I will change it a bit